universal-login-commons/lib/core/models/message.ts

```diff
@@ -35,14 +35,14 @@ export type MessageStatus = {
   collectedSignatures: string[],
   totalCollected: number,
   required: number,
-  state: MessageState
+  state: RepositoryItemState
 };
 
 export type MessageWithFrom = PartialRequired<SignedMessage, 'from'>;
 
 export type MessageWithoutFrom = Omit<SignedMessage, 'from'>;
 
-export type MessageState = 'AwaitSignature' | 'Queued' | 'Pending' | 'Error' | 'Success';
+export type RepositoryItemState = 'AwaitSignature' | 'Queued' | 'Pending' | 'Error' | 'Success';
```

Comment:
I am not sure about the nameing and type.
I would propose sth in this direction:
```ts
export type MinableState = MessageState | DeploymentState
```

Reply:
Done.